modules/op-customizing-pipelines-as-code-configuration.adoc

```diff
@@ -19,12 +19,30 @@ To customize {pac}, cluster administrators can configure the following parameter
 
 | `max-keep-days` | The number of the days for which the executed pipeline runs are kept in the `pipelines-as-code namespace`. 
 
-Note that this configmap setting does not affect the cleanups of a user's pipeline runs, which are controlled by the annotations on the pipeline run definition in the user's GitHub repository. |
+Note that this configmap setting does not affect the cleanups of a user's pipeline runs, which are controlled by the annotations on the pipeline run definition in the user's GitHub repository. |  NA
 
 | `secret-auto-create` | Indicates whether or not a secret should be automatically created using the token generated in the GitHub application. This secret can then be used with private repositories. | `enabled` 
 
 | `remote-tasks` | When enabled, allows remote tasks from pipeline run annotations. | `enabled`
 
 | `hub-url` | The base URL for the link:https://api.hub.tekton.dev/v1[Tekton Hub API]. | `https://hub.tekton.dev/`
 
+| `hub-catalog-name` | The Tekton Hub catalog name. | `tekton`
+
+| `tekton-dashboard-url` | The URL of the Tekton Hub dashboard. Pipelines as Code uses this URL to generate a `PipelineRun` URL on the Tekton Hub dashboard.  | NA
+
+| `bitbucket-cloud-check-source-ip` | Indicates whether to secure the service requests by querying IP ranges for a public bitbucket. Changing the parameter's default value might result into a security issue. | `enabled`
+
+| `bitbucket-cloud-additional-source-ip` | Indicates whether to provide additional set of IP ranges or networks, which are separated by commas. | NA
+
+| `max-keep-run-upper-limit` | A maximum limit for the `max-keep-run` value for a pipeline run. | NA
+
+| `default-max-keep-runs` | A default limit for the `max-keep-run` value for a pipeline run. If defined, the value is applied to all pipeline runs, which do not have a `max-keep-run` annotation. | NA
```

Comment:
"applied to all pipeline runs, which do not have ..."

I would probably say ""applied to all pipeline runs that do not have ..." as it is restrictive